### ISSUE TITLE
Requires login

### ISSUE THREAD

**Issue by GerryOnGithub** [marklagendijk/node-pm2-windows-startup#15]
After a reboot it seems to be necessary to login for anything to happen. Note the registry entry is under CURRENTUSER. But servers should not and cannot rely on a user login after reboot for stuff to run.

**Comment by joelhickok** [comment]
Yes, because of this issue, this is only a practical extension for PM2 if used on on a machine that is regularly accepting a logged in user.

**Comment by harvzor** [comment]
I found that logging in did not seem to get it to run on its own either.

**Comment by mcdado** [comment]
If running on a server, you could consider using [automatic logon](https://docs.microsoft.com/en-us/troubleshoot/windows-server/user-profiles-and-logon/turn-on-automatic-logon) (also [here is a easier way](https://hideez.com/blogs/news/login-without-password-in-windows) to do it through UI and not directly in the registry). Basically you save the credentials in the Windows Registry and allow the computer to automatically login.

Of course make sure your host is in a secure environment and any remote access is limited to a VPN or something like it.
